Source/doom.cpp

```diff
@@ -9,10 +9,9 @@
 
 namespace devilution {
 
-int doom_quest_time;
-int doom_stars_drawn;
-std::unique_ptr<uint8_t[]> doomCel;
-bool doomflag;
+int DoomQuestTime;
+std::unique_ptr<uint8_t[]> DoomCel;
+bool DoomFlag;
 int DoomQuestState;

```

Comment:
thanks, this will make the global cleanup seam less drastic :D

Reply:
No problem.